samples/aspnetcore/WebAppPlugin2/WebAppPlugin2.csproj

```diff
@@ -1,11 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Can you keep this at netstandard2.0? It was an intentional choice. I'm find if we convert plugin1 to netcoreapp2.0, but I want to keep netstandard in the sample.